webpack.renderer.js

```diff
@@ -3,7 +3,7 @@ const webpack = require('webpack');
 const { merge } = require('webpack-merge');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const AngularCompilerPlugin = require('@ngtools/webpack').AngularCompilerPlugin;
+const AngularWebpackPlugin = require('@ngtools/webpack').AngularWebpackPlugin;
```

Comment:
Should be importing like this
```suggestion
const { AngularWebpackPlugin } = require('@ngtools/webpack');
```
due to this note on the export
```
/**` @deprecated Deprecated as of v12, please use the direct exports
 * (`AngularWebpackPlugin` instead of `ivy.AngularWebpackPlugin`)
 */```